modules/mod_ginger_base/lib/js/search-suggestions.js

```diff
@@ -47,9 +124,24 @@
 
             }
 
+            $(document).on('keyup', function(e) {
```

Comment:
This interferes with all Enter keypresses in _any_ element on the page, which causes bugs when Enter is pressed an element other than `search_suggestions`: 

```javascript
Uncaught TypeError: this._navigateToSearchPage is not a function
    at t.(anonymous function).(anonymous function).update (http://zuiderzeecollectie.docker.dev/lib/bootstrap/js/bootstrap.min~/js/qlobber~pubzub~modules/z.live~/js/vendors/jquery.fancybox.min~jquery.fancybox-media~/js/base~global-nav~search-suggestions~content-group-nav~foldout~foundation~carousel~map-location~parallax~anchor~expand~vendors/flowplayer-3.2.12.min~slick.min~purl~/js/search/search_ui~search~components/input-text~sort~pager~types~/js/mail-decode~masthead-map~1921967954.js:1141:26)
    at t.(anonymous function).(anonymous function).update (http://zuiderzeecollectie.docker.dev/lib/js/apps/jquery-latest.min~jquery-migrate-1.2.1.min~jquery-ui-latest.min~/js/modules/jquery.ui.touch-punch.min~1545675240.js:11:5029)
    at HTMLDocument.<anonymous> (http://zuiderzeecollectie.docker.dev/lib/bootstrap/js/bootstrap.min~/js/qlobber~pubzub~modules/z.live~/js/vendors/jquery.fancybox.min~jquery.fancybox-media~/js/base~global-nav~search-suggestions~content-group-nav~foldout~foundation~carousel~map-location~parallax~anchor~expand~vendors/flowplayer-3.2.12.min~slick.min~purl~/js/search/search_ui~search~components/input-text~sort~pager~types~/js/mail-decode~masthead-map~1921967954.js:1221:24)
    at HTMLDocument.dispatch (http://zuiderzeecollectie.docker.dev/lib/js/apps/jquery-latest.min~jquery-migrate-1.2.1.min~jquery-ui-latest.min~/js/modules/jquery.ui.touch-punch.min~1545675240.js:3:8436)
    at HTMLDocument.r.handle (http://zuiderzeecollectie.docker.dev/lib/js/apps/jquery-latest.min~jquery-migrate-1.2.1.min~jquery-ui-latest.min~/js/modules/jquery.ui.touch-punch.min~1545675240.js:3:5139)
```